script/runc.spec

```diff
@@ -0,0 +1,64 @@
+##  Build runc RPM.
+##   1. Copy runc project tarball into ~/rpmbuild/BUILD/runc.tar.gz
+##   2. run rpmbuild -ba script/runc-centos.spec  -> will build source and binary package both.
+##
+##  Note:
+##   1. tarball must be a git project and will checkout the version while building rpms.
+##   2. this spec will build runc in dynamic link not static (if static is need, just uncomment build rules)
+##   3. need to update the TAG of runc project if release a rpm package.
+##
+%global _sbindir /usr/local/sbin
+%global _mandir  /usr/local/share/man
+%global _docdir  /usr/local/share/doc
+
+Name: runc
+Version: %{_version}
+Release: %{_release}
+Summary: A tool for managing OCI containers
+
+License: ASL 2.0
+Source: %{name}-%{version}.tar.gz
+
+URL: https://www.opencontainers.org/
+Vendor: OCI
+Packager: OCI 
+
+BuildRequires: gcc-go
+BuildRequires: glibc
+BuildRequires: make
+BuildRequires: libseccomp-devel 
+BuildRequires: libselinux-devel
+BuildRequires: libapparmor-devel
+BuildRequires: go-md2man
```

Comment:
hi @lsm5  ,  I will release runC rpms based on redhat, if rpm spec was integrated into runc project, it would be  much more convenient.  

For  libapparmor-devel pacakge,  I found it in [https://www.rpmfind.net/](url)